docs/src/content/docs/installation.mdx

```diff
@@ -41,6 +41,14 @@ You can follow the installation instructions in the
       pesde should now be installed on your system. You may need to restart your
       computer for the changes to take effect.
 
+      [dev-mode-guide]: https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development
+
+      <Aside type="caution" title="Warning">
+      pesde uses symlinks which are an administrator-level operation on Windows.
+      To ensure proper functionality, enable [Developer Mode][dev-mode-guide].<br/><br/>
```

Comment:
```suggestion
      To ensure proper functionality, enable [Developer Mode](https://learn.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development).
      
      
```

Let's inline this URL and use newlines instead of `<br>`s.